README.md

```diff
@@ -5,5 +5,35 @@ This is a sample of a Java Spring app that works with Tilt and the Tanzu Applica
 ## Dependencies
 1. [kubectl CLI](https://kubernetes.io/docs/tasks/tools/)
 1. [Tilt version >= v0.22.6](https://docs.tilt.dev/install.html)
-1. [Tanzu CLI apps plugin v0.2.0](https://github.com/vmware-tanzu/tanzu-cli-apps-plugins)
-1. A cluster with Tanzu Application Platform, the Inner Loop Supply Chain, and their dependencies. Download these from Tanzunet.
+1. Tanzu CLI and the apps plugin v0.2.0 which are provided as part of [Tanzu Application Platform](https://network.tanzu.vmware.com/products/tanzu-application-platform)
+1. A cluster with Tanzu Application Platform, and one of the provided supply chains, plus its dependencies. These supply chains are part of [Tanzu Application Platform](https://network.tanzu.vmware.com/products/tanzu-application-platform).
```

Comment:
We have only tested and can support that debug / live-reload works with the Default Supply Chain, not any supply chain.

Reply:
OK, I have updated the readme and also added Apache License 2.0